styles/stylequery.css

```diff
@@ -59,20 +53,203 @@
+
+  .project2 {
+    margin-left: 0;
+  }
+
+  .project3 {
+    margin-right: 0;
+    width: 48vw;
+    height: 320px;
+    background-size: cover;
+  }
+
+  .project4 {
+    margin-left: 0;
+    width: 30vw;
+    height: 300px;
+    background-size: cover;
+  }
+
+  .proyect-foot1 {
+    display: none;
+  }
+
+  .proyect-foot2 {
+    display: none;
+  }
+
+  .proyect-foot3 {
+    display: none;
+  }
+
+  .proyect-foot4 {
+    display: none;
+  }
+
+  .proyect-foot5 {
+    display: none;
+  }
+
+  .proyect-foot6 {
+    display: none;
+  }
+
+  .picture6 {
+    grid-area: picture6;
+
+    /* display: block; */
+    background-image: url(/pict-mobile/Rectangle\ 21\ \(5\) .svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  .ornament1 {
+    grid-area: ornament1;
+    display: block;
+    background-image: url(/pic-desktop/image-geometry_5.svg), url(/pic-desktop/image-geometry_4.svg);
+    background-repeat: no-repeat no-repeat;
+    background-size: contain contain;
+    background-size: 124px 124px 87px 87px;
+    background-position: left top, right top;
+  }
+
+  .ornament2 {
+    grid-area: ornament2;
+    display: block;
+    background-image: url(/pic-desktop/Shape.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-size: 130px 130px;
+    background-position: left bottom;
+  }
+
+  .ornament3 {
+    grid-area: ornament3;
+    display: block;
+    background-image: url(/pic-desktop/image-geometry_2.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-size: 430px 280px;
+    background-position: right bottom;
+    padding: 200px 0 120px 300px;
+    margin: -200px 0 -120px -300px;
+  }
+
+  .ornament4 {
+    grid-area: ornament4;
+    display: block;
+    background-image: url(/pic-desktop/Group\ 83.svg);
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-size: 350px 175px;
+    background-position: left top;
+    padding: 0 350px 200px 0;
+    margin: 240px -350px -200px 180px;
+  }
+
+  .works-cards1 {
+    grid-area: works-cards1;
+  }
+
+  .works-cards1 .proyect-foot1 {
+    display: none;
+  }
+
+  .works-cards1:hover .proyect-foot1 {
+    display: block;
+  }
+
+  .works-cards2 {
+    grid-area: works-cards2;
+  }
+
+  .works-cards2 .proyect-foot2 {
+    display: none;
+  }
+
+  .works-cards2:hover .proyect-foot2 {
+    display: block;
+    width: 400px;
+  }
+
+  .works-cards3 {
+    grid-area: works-cards3;
+  }
+
+  .works-cards3 .proyect-foot3 {
+    display: none;
+  }
+
+  .works-cards3:hover .proyect-foot3 {
+    display: block;
+  }
+
+  .works-cards4 {
+    grid-area: works-cards4;
+  }
+
+  .works-cards4 .proyect-foot4 {
+    display: none;
+  }
+
+  .works-cards4:hover .proyect-foot4 {
+    display: block;
+  }
+
+  .works-cards5 {
+    grid-area: works-cards5;
+  }
+
+  .works-cards5 .proyect-foot5 {
+    display: none;
+  }
+
+  .works-cards5:hover .proyect-foot5 {
+    display: block;
+  }
+
```

Comment:
- [ ] Please make sure that you use media queries for the Desktop breakpoint in the CSS file. (the value for the `min-width` property should be in the range of `768px` or wider)